src/group_theory/subgroup.lean

```diff
@@ -889,6 +938,9 @@ begin
     refine ⟨-n, neg_gsmul x n⟩ }
 end
 
+lemma closure_singleton_zero : closure ({0} : set A) = ⊥ :=
```

Comment:
I guess it can't be deduced from `closure_singleton_one` through `to_additive`? Even if it can't, it's good to register (after the fact) a `to_additive` attribute so that later proofs can be converted automatically.